### ISSUE TITLE
[Broken Website] bestbuy.com

### ISSUE THREAD

**Issue by Gitoffthelawn** [darkreader/darkreader#7729]
# Broken Website Report

## Website Address
https://www.bestbuy.com/site/hydrow-connected-rower-silver-black/6349559.p?skuId=6349559

## Steps To Reproduce
- Launch Firefox with DR in dynamic dark mode
- Go to https://www.bestbuy.com/site/hydrow-connected-rower-silver-black/6349559.p?skuId=6349559
- Notice that some elements flash white, go black, then flash white, then go black again
- Notice that some elements (e.g. TotalTech logo) are black on black

## Expected Behavior
No flashing, no black on black

## Actual Behavior
See above

## System Information
- OS: Linux
- Browser: Firefox 95.0.2
- Dark Reader Version: 4.9.43

## Additional Context
I started creating some fixes for this, but I wasn't sure of the best techniques to use, so I'll leave it to others.

Note that this issue is on most (all?) of their product pages.  So if this product isn't available, you can just try a different product.

**Comment by Gusted** [comment]
> * Notice that some elements flash white, go black, then flash white, then go black again

So this kind of concerns me as the logo's etc should be fixed by fixes. With most of the "flashing elements", they are some kind of element that is also watched by the site and darkreader and the site gets into conflict. Yet this one doesn't seem to be of that kind, it seems to me that the CSS is loaded on the fly(as you scroll !?!? f*ck weird behavior if you ask me), and dark reader need a few ms(when the render engine already renders the site) in order to modify the correct CSS for it.

Regards,
Gusted

**Comment by Gitoffthelawn** [comment]
Yes, I don't know why so many web developers choose to do things the stupid way.  ;)

It's like everyone wants to reinvent the wheel because they think they are smarter than the person who invented the wheel.  The wheel works great and the person who invented it is smarter than all of us... leave it the f*ck alone and accept you're a dumb shit like the rest of us.  :)  Egos and narcissism today...  never seen anything like it before.
